images/node/Dockerfile-debian

```diff
@@ -0,0 +1,51 @@
+###############################################################
+# DOCKER IMAGE - Debian based Nodejs Application (Express.js) #
+###############################################################
+
+FROM debian:latest
```

Comment:
Lets use this base image instead. Comes pre-installed with node and a few other useful libs. 

https://hub.docker.com/_/node/

node:20-book-slim